department-of-veterans-affairs/vets-website
# 48682 - Update the VAOS FE code to use Drupal Source of Truth for Cerner information

## Description
This PR implements changes to use the Drupal as the source of truth for determining if a veteran is a Cerner user.

**How to test**

- Turn on feature flag - vaOnlineSchedulingUseDsot
- Modify ../vaos/services/mocks/index.js:409 to set facility id = 442
- Schedule an appointment
- Verify link to route user to Cerner exist on 'Choose a VA location' page

**NOTE**: 'is_cerner' flag in index.js is not used since Cerner information is coming from Drupal Source of Truth. Also, real facility ids are used.

## Original issue(s)
fixes department-of-veterans-affairs/va.gov-team#48682


## Testing done
Unit testing

## Screenshots
![localhost_3001_health-care_schedule-view-va-appointments_appointments_ (4)](https://user-images.githubusercontent.com/3587066/200369583-ec7a9a9a-fe48-43c4-98df-2e1bb28816af.png)


## Acceptance criteria
- [ ] All test should pass

## Definition of done
